lib/engine/game/base.rb

```diff
@@ -2052,8 +2052,12 @@ def priority_deal_player
         end
       end
 
-      def next_sr_position(entity)
-        player_order = @round.current_entity&.player? ? @round.pass_order : @players
+      def next_sr_position(entity, order)
+        player_order = if @round.current_entity&.player?
+                         order == :first_to_pass ? @round.pass_order : []
```

Comment:
Is this necessary? I believe pass order is [] unless used.

Reply:
At my computer now and pass_order is not [].

Reply:
I want to use passed players only with first-pass. With most_cash I think it would be best to never show the next priority in SR. 

I also could just plain return nil in that case. Maybe that would be clearer

Reply:
Its fine as is. I marked this comment as resolved.